### PR TITLE
use fastSim era to apply FastSim mods on TrackValidation_cff

### PR DESCRIPTION
**Pull request by lveldere**
After a discussion in the simulation meeting of Aug 21,
a new era 'fastSim' was introduced by Mark Grimes #10906 .
The plan is to use this fastSim era to apply FastSim mods on the configuration 
ofValidation sequences and modules.

This pull request uses the fastSim era to apply FastSim mods on the configuration of the Tracking Validation.

To minimise the number of FastSim mods,
some copy pasted psets were replaced with clones in the file
Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py

In addition, an obsolete, outdated cfg file is removed:
FastSimulation/Validation/python/trackingParticlesFastSim_cfi.py

@makortel 
please let me know if you have any suggestions / questions

@deguio and other DQM responsibles
Does this need a discussion between the FastSim and the DQM group?
The plan would be to gradually migrate the whole validation sequence.

### CODE REVIEW

**Comment by lveldere** [comment]
please test

File: Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py [hunk 1]
```diff
@@ -14,73 +14,13 @@
...
-    intimeOnly = cms.bool(False),
-    stableOnly = cms.bool(False),
-    minRapidity = cms.double(-2.5),
-    minHit = cms.int32(0),
-    ptMin = cms.double(0.9),
-    maxRapidity = cms.double(2.5),
-    lip = cms.double(30.0),
-    tip = cms.double(3.5)
-)
+# TODO: clone all these guys from the first one
```
Comment (makortel) [inline Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py]: Thanks :) But isn't the comment obsolete? (i.e. the todo is already done)

Comment (lveldere) [inline Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py]: yep :-)

On Mon, Aug 24, 2015 at 2:52 PM, Matti Kortelainen <notifications@github.com

**Comment by makortel** [comment]
@lveldere Looks good to me. I only want to note that now also the three other MTV instances (`trackValidatorFromPV`, `trackValidatorFromPVAllTP`, `trackValidatorAllTPEffic`) will be run for FastSim. Probably that is fine (or even desired), but I just wanted to note it explicitly.

**Comment by lveldere** [comment]
@makortel 
This is indeed by intention.
Things run without crashes, but I did not check whether the output of these new instances makes sense.

**Comment by lveldere** [comment]
+1

comparisons look fine

**Comment by civanch** [comment]
+1

**Comment by lveldere** [comment]
@danduggan @franzoni @deguio @davidlange6 

Would you mind reviewing and approving for operations and DQM?
If any further discussion is needed, please inform me.
Cheers
Lukas

**Comment by danduggan** [comment]
+1

**Comment by danduggan** [comment]
@lveldere I've approved this, but in terms of a full scale migration for the fastsim validation sequence, it would be good to have this discussion at our next DQM/pdmv meeting (sept 1st).

**Comment by lveldere** [comment]
@danduggan

okay, great.
I'll prepare a couple of slides for Tuesday,
to initiate the discussion.

**Pull request merged**